### PR TITLE
Model cleanup

### PR DESCRIPTION
**Pull request by fergusdixon**
## Overview

| Q                 | A
| ----------------- | ---
| Bugfix?           | yes
| New feature?      | no
| Breaking Changes? | no
| Fixed issues      | #

## The problem
I had left a few properties out of my earlier mocks, so I updated them.
...

## How I resolved it
Updated the mocks
...

Notify the following people: @darrynten

### CODE REVIEW

File: tests/SageOne/Models/AssetCategoryModelTest.php [hunk 1]
```diff
@@ -53,7 +54,7 @@ public function testAttributes()
                 'type' => 'DateTime',
                 'nullable' => true,
                 'readonly' => true,
-            ],
+            ]
```
Comment (darrynten) [inline tests/SageOne/Models/AssetCategoryModelTest.php]: This trailing comma should remain :)

File: tests/SageOne/Models/SupplierBankDetailModelTest.php [hunk 2]
```diff
@@ -111,9 +125,9 @@ public function testGetId()
             $this->assertEquals(1, $model->id);
             $this->assertEquals(2, $model->supplierId);
             $this->assertEquals('sample string 3', $model->bankAccountHolder);
-            $this->assertEquals('sample string 4', $model->bankAccountNumber);
+            $this->assertEquals('sample strin2', $model->bankAccountNumber);
             $this->assertEquals('sample string 5', $model->bankName);
-            $this->assertEquals('sample string 6', $model->bankBranchCode);
+            $this->assertEquals('123456', $model->bankBranchCode);
```
Comment (darrynten) [inline tests/SageOne/Models/SupplierBankDetailModelTest.php:130]: So their types don't even match

**Review by darrynten — CHANGES_REQUESTED** [review]
One comma

File: tests/SageOne/Models/SupplierBankDetailModelTest.php (hunk 2 above)
Comment (fergusdixon) [reply tests/SageOne/Models/SupplierBankDetailModelTest.php:130]: No its still a string

**Review by darrynten — APPROVED** [review]
Looks good, feel free to merge once all checks are green

**Pull request merged**